### ISSUE TITLE
Links not rendering when extending

### ISSUE THREAD

**Issue by bettsz** [valentine195/fantasy-statblocks#149]
When creating a new statblock by extending an existing one links that worked in the original are shown as plain text not rendered as a link in the extended statblock.
Original
![Screenshot_20230301-175741_Obsidian](https://user-images.githubusercontent.com/9614015/222311108-f38c8c9e-aa7f-4e4b-a67b-209642477793.jpg)
Extended
![Screenshot_20230301-175902_Obsidian](https://user-images.githubusercontent.com/9614015/222311270-f64a4e12-596a-4e6a-9064-2987a247b81b.jpg)

**Comment by bettsz** [comment]
@valentine195 Thanks for the speedy update, however now it looks like the issue is reversed, my original stat block doesn't render links while my extended ones do. I assume because it looks like the tranform function you moved the JSON.parse stuff into only runs for extensions. Was there a conflict with having the JSON.parse you commented out in the build function also? Sorry for being a pain.

**Comment by valentine195** [comment]
@bettsz hmm mine work fine:
<img width="676" alt="image" src="https://user-images.githubusercontent.com/38669521/222442158-c4579c65-49da-4b1e-873c-ce4c0bb1f2d3.png">

can you show me / paste your underlying code block?

**Comment by bettsz** [comment]
I was able to replicate what you have, but I've primarily been setting these up in the frontmatter so it looks like that's where the issue is.
![statblock_test](https://user-images.githubusercontent.com/9614015/222484665-40879d96-fde1-4ad2-8ba1-3a989cbc014d.PNG)

**Comment by bettsz** [comment]
@valentine195 I seemed to have fixed this by changing the transform function a bit so that it doesn't return if it's not an extension and allows either to go through the JSON.parse. I set up a [fork](https://github.com/bettsz/fantasy-statblocks) and was able to test this locally but I'm fumbling with the GitHub workflows. Anyway this seems to be working for me right now, I'm happy to try and get my fork working correctly so I can do a pull request but it'll take me a bit to figure out the workflow issue.
